sdk/servicebus/azure-messaging-servicebus-stress/src/main/java/com/azure/messaging/servicebus/stress/scenarios/MessageSender.java

```diff
@@ -23,48 +22,33 @@
 public class MessageSender extends ServiceBusScenario {
     private static final ClientLogger LOGGER = new ClientLogger(MessageSender.class);
 
-    private static final Random RANDOM = new Random();
+    @Value("${DURATION_IN_MINUTES:15}")
+    private int durationInMinutes;
 
-    @Value("${SEND_TIMES:100000}")
-    private int sendTimes;
+    @Value("${SEND_MESSAGE_RATE:100}")
+    private int sendMessageRatePerSecond;
 
-    @Value("${SEND_MESSAGES:10}")
-    private int messagesToSend;
+    @Value("${BATCH_SIZE:2}")
+    private int batchSize;
 
-    @Value("${PAYLOAD_SIZE_IN_BYTE:8}")
-    private int payloadSize;
+    @Value("${MESSAGE_SIZE_IN_BYTES:8}")
+    private int messageSize;
```

Comment:
Should this test support concurrency similarly to the async scenario by using a ExecutorService or ForkJoinPool?

Reply:
it probably should, but I'd prefer to do it separately when focusing on sync concurrency validation.